packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/SMS/index.js

```diff
@@ -35,6 +40,16 @@ class SmsAuthContainer extends React.PureComponent {
       this.props.goBackOnSuccess()
     }
   }
+  handleMount () {
+    this.setState({
+      changeNumberToggled: !this.state.changeNumberToggled
+    })
+  }
+  handleUpdate () {
```

Comment:
this method isnt used, remove.

Reply:
Also line 37 with `updateUI`, needs to be updated/removed

Reply:
replaced updateUI with handleUpdate